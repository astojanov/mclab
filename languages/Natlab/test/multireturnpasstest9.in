
[a, b(a), a, c(b(a))] = foo();
