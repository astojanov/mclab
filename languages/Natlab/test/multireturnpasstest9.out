
[a, t1, t2, t3] = foo();
b(a) = t1;
a = t2;
c(b(a)) = t3;
